### PR TITLE
Workflow that add merged PRs to release draft

### PR DESCRIPTION
**Pull request by StefanIvemo**
Created a workflow that add details about each merged PR to a release draft.

- Runs on push to main (will only happen when a PR is merged)
- Verifies if a release draft exists or not.
- Get PR number form commit message and gathers info about the merged PR.
- Verifies if labels `bugFix`, `newFeature` or `updatedDocs` are added, if not the PR will not be added to release draft.
- Update/Create release notes in JSON format to allow us to work with the data when creating a new release.

NOTE: Will add check to PRs that require labels to be specified.

### CODE REVIEW

**Comment by JohnRoos** [comment]
I haven't had time to check the code yet, but wouldn't it be a good idea to include the Enhancement tag as well?

File: scripts/mergedPRDetailsToReleaseDraft.ps1 [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    [string]$Token
+)
+
+$header = @{
+    "authorization" = "token $Token"
+    "Accept"        = "application/vnd.github.v3+json"
+}
+
+#Get all releases including drafts
+$getReleases = Invoke-RestMethod -Method Get -Headers $header -URI  "https://api.github.com/repos/StefanIvemo/ActionsTest/releases"
```
Comment (bjompen) [inline scripts/mergedPRDetailsToReleaseDraft.ps1:14]: This URI points to a different repo than this. is this correct?

Comment (StefanIvemo) [reply scripts/mergedPRDetailsToReleaseDraft.ps1:14]: 😅 Ooops!

Comment (bjompen) [reply scripts/mergedPRDetailsToReleaseDraft.ps1]: Looks better, but I would instead suggest setting the URI as a variable and referencing that one. That way, if the module ever changes home it's way easier to change this as well.

Comment (StefanIvemo) [reply scripts/mergedPRDetailsToReleaseDraft.ps1]: I'll fix

**Review by JohnRoos — APPROVED** [review]

**Pull request merged**